scripts/west_commands/run_common.py

```diff
@@ -134,9 +136,21 @@ def add_parser_common(command, parser_adder=None, parser=None):
     group.add_argument('--board-dir', metavar='DIR', help='board directory')
     # FIXME: we should just have a single --file argument. The variation
     # between runners is confusing people.
-    group.add_argument('--elf-file', metavar='FILE', help='path to zephyr.elf')
-    group.add_argument('--hex-file', metavar='FILE', help='path to zephyr.hex')
-    group.add_argument('--bin-file', metavar='FILE', help='path to zephyr.bin')
+    group.add_argument('--elf-file',
+                       metavar='FILE',
+                       action=partial(depr_action,
+                                      replacement='-f/--file'),
+                        help='path to zephyr.elf')
```

Comment:
Putting this here implies:

```
$ west debug -r jlink --elf-file build/zephyr/some-other.elf
Argument --elf-file is deprecated, use -f/--file instead.
```

but I don't think that's actually what you meant to happen; is it? Especially since the jlink runner's debug code doesn't support `--file`, but you can currently pass `--elf-file` to override the default.

Reply:
added `--file` handling to jlink's debug code